src/main/java/com/github/susom/dbgoodies/etl/Etl.java

```diff
@@ -15,295 +15,364 @@
-          break;
-        case Types.NVARCHAR:
-        case Types.VARCHAR:
-        case Types.CHAR:
-        case Types.NCHAR:
-          record.put(colName, rs.getStringOrNull(colName));
-          break;
-        default:
-          record.put(colName, rs.getStringOrNull(colName));
-      }
-    }
-
-    /**
-     * Actually begin the writing AVRO file.
-     */
-    public void start() {
-      select.fetchSize(batchSize).query(rs -> {
-
-        /* initialize susom database library elements */
-        String[] names = null;
-        ResultSetMetaData rsMeta = null;
-        int type = TYPE_NONE;
-
-        /* initialize apache AVRO library elements */
-        String avroSchemaText = null;
-        org.apache.avro.Schema schema = null;
-        GenericRecord record = null;
-        DatumWriter<GenericRecord> datumWriter = null;
-        DataFileWriter<GenericRecord> dataFileWriter = null;
-        AvroSchema avroSchema = null;
-
-        /* initialize jaxson (json) library elements */
-        ObjectMapper mapper = new ObjectMapper();
-        while (rs.next()) {
-          if (rsMeta == null) {
-            /* Get table metadata */
-            rsMeta = rs.getMetadata();
-
-            /* Get column names */
-            names = rs.getColumnLabels();
-
-            /* define AVRO schema object */
-            avroSchema = AvroSchema.getInstance();
-            avroSchema.setName(tableName);
-
-            /* Iterate through columns and add AVRO schema to object */
-            for (int i = 0; i < names.length; i++) {
-              AvroRecord recordJson = AvroRecord.getIntance();
-              recordJson.setName(names[i]);
-              type = rsMeta.getColumnType(i + 1);
-              recordJson.getType().add(AvroRecord.getType(type));
-              avroSchema.getFields().add(recordJson);
-            }
-            /* create avro json schema string */
-            avroSchemaText = mapper.writeValueAsString(avroSchema);
-
-            /* define AVRO schema */
-            schema = new org.apache.avro.Schema.Parser().parse(avroSchemaText);
-            log.debug("AvroSchema for table '%s': %s", tableName, avroSchemaText);
-
-            /* add AVRO schema to records and avro file */
-            datumWriter = new GenericDatumWriter<GenericRecord>(schema);
-            dataFileWriter = new DataFileWriter<GenericRecord>(datumWriter);
-            dataFileWriter.create(schema, new File(path));
-          }
-
-          record = new GenericData.Record(schema);
-          for (int i = 0; i < names.length; i++) {
-            type = rsMeta.getColumnType(i + 1);
-            /* Put the values into the avro record */
-            updateAvroRecord(record, type, rs, names[i]);
-          }
-          /* Append record to avro file */
-          dataFileWriter.append(record);
-        }
-
-        dataFileWriter.close();
-        return null;
-      });
-    }
-  }
+	private static final Logger log = LoggerFactory.getLogger(Etl.class);
```

Comment:
Can you set your Eclipse code style to use the Google standards? In particular, alphabetical imports, two spaces (no tabs).